idseq_dag/steps/run_alignment_remotely.py

```diff
@@ -439,6 +455,7 @@ def aws_cp_operation(input_fa):
                             self.__copy_multihit_remote_outfile(
                                 key_path, remote_username, instance_ip,
                                 multihit_remote_outfile, multihit_local_outfile)
+                            self.__delete_remote_dir(remote_work_dir, key_path, remote_username, instance_ip)
```

Comment:
It's usually better to put cleanup in the "finally" section, as you might want it to run even in case the chunk crashes.  Unless it is your express intention to leave around crashed chunks for debugging.

Reply:
The `finally` section on line 495 would be good.